types/node/buffer.d.ts

```diff
@@ -530,7 +530,7 @@ declare module 'buffer' {
              * @param [fill=0] A value to pre-fill the new `Buffer` with.
              * @param [encoding='utf8'] If `fill` is a string, this is its encoding.
              */
-            alloc(size: number, fill?: string | Buffer | number, encoding?: BufferEncoding): Buffer;
+            alloc(size: number, fill?: string | Buffer | Uint8Array | number, encoding?: BufferEncoding): Buffer;
```

Comment:
We can remove `Buffer` then since `Buffer` inherits from `Uint8Array`

Reply:
So there is no advantage in explicit setting the types?

Reply:
I mean, this is a type checker, not a "documentation checker". But it would be nice to have all the argument available types stated when a wrong type is used, isn't?

Reply:
*stated in a explicit way